packages/check-es-compat/bin/cli.mjs

```diff
@@ -46,3 +47,41 @@ async function execute(files) {
 
   return { hasErrors: results.some((result) => result.errorCount > 0) };
 }
+
+function parseArguments(args) {
```

Comment:
I'm hesitant to put this code in - there are a few different branches and checks so it's not straightforward to have confidence whether it covers everything it needs to.

I think it should use minimist.

Also there are likely to be many polyfills so perhaps it would be better instead to take a `--polyfills-file` that is a path to a file with one polyfill on each line.

Reply:
Yeah, ideally it should use minimist, but I decided not to do it because that would likely be a lot more work. And I'm not sure you'd like the idea to add new dependencies, etc.

In any case, I implemented this for our project and I thought I'd contribute it rather than just opening an issue. If you think this is not the right direction for the library, I think it's better if you close this PR and work on this yourself (or I can open a normal issue to keep track of the missing feature). But I don't think I'll take it further, because this simple approach is enough for our use-case and we're using the patch with [patch-package](https://www.npmjs.com/package/patch-package).

Reply:
Fair, thanks. I've created issue #93 for it, but am not planning to work on it.